---a/net/minecraft/entity/monster/EntityBlaze.java
+++b/net/minecraft/entity/monster/EntityBlaze.java
@@ -18,17 +18,14 @@
 import net.minecraft.util.EnumParticleTypes;
 import net.minecraft.util.MathHelper;
 import net.minecraft.world.World;
-import net.minecraftforge.fml.relauncher.Side;
-import net.minecraftforge.fml.relauncher.SideOnly;
 
-public class EntityBlaze extends EntityMob
-{
+
+public class EntityBlaze extends EntityMob {
+
     private float heightOffset = 0.5F;
     private int heightOffsetUpdateTime;
     private static final String __OBFID = "CL_00001682";
 
-    public EntityBlaze(World worldIn)
-    {
+    public EntityBlaze(World worldIn) {
         super(worldIn);
         this.isImmuneToFire = true;
         this.experienceValue = 10;
@@ -41,88 +41,67 @@
         this.targetTasks.addTask(2, new EntityAINearestAttackableTarget(this, EntityPlayer.class, true));
     }
 
-    protected void applyEntityAttributes()
-    {
+    protected void applyEntityAttributes() {
         super.applyEntityAttributes();
         this.getEntityAttribute(SharedMonsterAttributes.attackDamage).setBaseValue(6.0D);
         this.getEntityAttribute(SharedMonsterAttributes.movementSpeed).setBaseValue(0.23000000417232513D);
         this.getEntityAttribute(SharedMonsterAttributes.followRange).setBaseValue(48.0D);
     }
 
-    protected void entityInit()
-    {
+    protected void entityInit() {
         super.entityInit();
-        this.dataWatcher.addObject(16, new Byte((byte)0));
+        this.dataWatcher.addObject(16, new Byte((byte) 0));
     }
 
-    protected String getLivingSound()
-    {
+    protected String getLivingSound() {
         return "mob.blaze.breathe";
     }
 
-    protected String getHurtSound()
-    {
+    protected String getHurtSound() {
         return "mob.blaze.hit";
     }
 
-    protected String getDeathSound()
-    {
+    protected String getDeathSound() {
         return "mob.blaze.death";
     }
 
-    @SideOnly(Side.CLIENT)
-    public int getBrightnessForRender(float p_70070_1_)
-    {
-        return 15728880;
-    }
-
-    public float getBrightness(float p_70013_1_)
-    {
+    public float getBrightness(float p_70013_1_) {
         return 1.0F;
     }
 
-    public void onLivingUpdate()
-    {
-        if (!this.onGround && this.motionY < 0.0D)
-        {
+    public void onLivingUpdate() {
+        if (!this.onGround && this.motionY < 0.0D) {
             this.motionY *= 0.6D;
         }
 
-        if (this.worldObj.isRemote)
-        {
-            if (this.rand.nextInt(24) == 0 && !this.isSilent())
-            {
+        if (this.worldObj.isRemote) {
+            if (this.rand.nextInt(24) == 0 && !this.isSilent()) {
                 this.worldObj.playSound(this.posX + 0.5D, this.posY + 0.5D, this.posZ + 0.5D, "fire.fire", 1.0F + this.rand.nextFloat(), this.rand.nextFloat() * 0.7F + 0.3F, false);
             }
 
-            for (int i = 0; i < 2; ++i)
-            {
-                this.worldObj.spawnParticle(EnumParticleTypes.SMOKE_LARGE, this.posX + (this.rand.nextDouble() - 0.5D) * (double)this.width, this.posY + this.rand.nextDouble() * (double)this.height, this.posZ + (this.rand.nextDouble() - 0.5D) * (double)this.width, 0.0D, 0.0D, 0.0D, new int[0]);
+            for (int i = 0; i < 2; ++i) {
+                this.worldObj.spawnParticle(EnumParticleTypes.SMOKE_LARGE, this.posX + (this.rand.nextDouble() - 0.5D) * (double) this.width, this.posY + this.rand.nextDouble() * (double) this.height, this.posZ + (this.rand.nextDouble() - 0.5D) * (double) this.width, 0.0D, 0.0D, 0.0D, new int[0]);
             }
         }
 
         super.onLivingUpdate();
     }
 
-    protected void updateAITasks()
-    {
-        if (this.isWet())
-        {
+    protected void updateAITasks() {
+        if (this.isWet()) {
             this.attackEntityFrom(DamageSource.drown, 1.0F);
         }
 
         --this.heightOffsetUpdateTime;
 
-        if (this.heightOffsetUpdateTime <= 0)
-        {
+        if (this.heightOffsetUpdateTime <= 0) {
             this.heightOffsetUpdateTime = 100;
-            this.heightOffset = 0.5F + (float)this.rand.nextGaussian() * 3.0F;
+            this.heightOffset = 0.5F + (float) this.rand.nextGaussian() * 3.0F;
         }
 
         EntityLivingBase entitylivingbase = this.getAttackTarget();
 
-        if (entitylivingbase != null && entitylivingbase.posY + (double)entitylivingbase.getEyeHeight() > this.posY + (double)this.getEyeHeight() + (double)this.heightOffset)
-        {
+        if (entitylivingbase != null && entitylivingbase.posY + (double) entitylivingbase.getEyeHeight() > this.posY + (double) this.getEyeHeight() + (double) this.heightOffset) {
             this.motionY += (0.30000001192092896D - this.motionY) * 0.30000001192092896D;
             this.isAirBorne = true;
         }
@@ -130,143 +130,117 @@
         super.updateAITasks();
     }
 
-    public void fall(float distance, float damageMultiplier) {}
+    public void fall(float distance, float damageMultiplier) {
+    }
 
-    protected Item getDropItem()
-    {
+    protected Item getDropItem() {
         return Items.blaze_rod;
     }
 
-    public boolean isBurning()
-    {
+    public boolean isBurning() {
         return this.func_70845_n();
     }
 
-    protected void dropFewItems(boolean p_70628_1_, int p_70628_2_)
-    {
-        if (p_70628_1_)
-        {
+    protected void dropFewItems(boolean p_70628_1_, int p_70628_2_) {
+        if (p_70628_1_) {
             int j = this.rand.nextInt(2 + p_70628_2_);
 
-            for (int k = 0; k < j; ++k)
-            {
+            for (int k = 0; k < j; ++k) {
                 this.dropItem(Items.blaze_rod, 1);
             }
         }
     }
 
-    public boolean func_70845_n()
-    {
+    public boolean func_70845_n() {
         return (this.dataWatcher.getWatchableObjectByte(16) & 1) != 0;
     }
 
-    public void func_70844_e(boolean p_70844_1_)
-    {
+    public void func_70844_e(boolean p_70844_1_) {
         byte b0 = this.dataWatcher.getWatchableObjectByte(16);
 
-        if (p_70844_1_)
-        {
-            b0 = (byte)(b0 | 1);
-        }
-        else
-        {
+        if (p_70844_1_) {
+            b0 = (byte) (b0 | 1);
+        } else {
             b0 &= -2;
         }
 
         this.dataWatcher.updateObject(16, Byte.valueOf(b0));
     }
 
-    protected boolean isValidLightLevel()
-    {
+    protected boolean isValidLightLevel() {
         return true;
     }
 
-    class AIFireballAttack extends EntityAIBase
-    {
+    class AIFireballAttack extends EntityAIBase {
+
         private EntityBlaze field_179469_a = EntityBlaze.this;
         private int field_179467_b;
         private int field_179468_c;
         private static final String __OBFID = "CL_00002225";
 
-        public AIFireballAttack()
-        {
+        public AIFireballAttack() {
             this.setMutexBits(3);
         }
 
-        public boolean shouldExecute()
-        {
+        public boolean shouldExecute() {
             EntityLivingBase entitylivingbase = this.field_179469_a.getAttackTarget();
             return entitylivingbase != null && entitylivingbase.isEntityAlive();
         }
 
-        public void startExecuting()
-        {
+        public void startExecuting() {
             this.field_179467_b = 0;
         }
 
-        public void resetTask()
-        {
+        public void resetTask() {
             this.field_179469_a.func_70844_e(false);
         }
 
-        public void updateTask()
-        {
+        public void updateTask() {
             --this.field_179468_c;
             EntityLivingBase entitylivingbase = this.field_179469_a.getAttackTarget();
             double d0 = this.field_179469_a.getDistanceSqToEntity(entitylivingbase);
 
-            if (d0 < 4.0D)
-            {
-                if (this.field_179468_c <= 0)
-                {
+            if (d0 < 4.0D) {
+                if (this.field_179468_c <= 0) {
                     this.field_179468_c = 20;
                     this.field_179469_a.attackEntityAsMob(entitylivingbase);
                 }
 
                 this.field_179469_a.getMoveHelper().setMoveTo(entitylivingbase.posX, entitylivingbase.posY, entitylivingbase.posZ, 1.0D);
-            }
-            else if (d0 < 256.0D)
-            {
+            } else if (d0 < 256.0D) {
                 double d1 = entitylivingbase.posX - this.field_179469_a.posX;
-                double d2 = entitylivingbase.getEntityBoundingBox().minY + (double)(entitylivingbase.height / 2.0F) - (this.field_179469_a.posY + (double)(this.field_179469_a.height / 2.0F));
+                double d2 = entitylivingbase.getEntityBoundingBox().minY + (double) (entitylivingbase.height / 2.0F) - (this.field_179469_a.posY + (double) (this.field_179469_a.height / 2.0F));
                 double d3 = entitylivingbase.posZ - this.field_179469_a.posZ;
 
-                if (this.field_179468_c <= 0)
-                {
+                if (this.field_179468_c <= 0) {
                     ++this.field_179467_b;
 
-                    if (this.field_179467_b == 1)
-                    {
+                    if (this.field_179467_b == 1) {
                         this.field_179468_c = 60;
                         this.field_179469_a.func_70844_e(true);
-                    }
-                    else if (this.field_179467_b <= 4)
-                    {
+                    } else if (this.field_179467_b <= 4) {
                         this.field_179468_c = 6;
-                    }
-                    else
-                    {
+                    } else {
                         this.field_179468_c = 100;
                         this.field_179467_b = 0;
                         this.field_179469_a.func_70844_e(false);
                     }
 
-                    if (this.field_179467_b > 1)
-                    {
+                    if (this.field_179467_b > 1) {
                         float f = MathHelper.sqrt_float(MathHelper.sqrt_double(d0)) * 0.5F;
-                        this.field_179469_a.worldObj.playAuxSFXAtEntity((EntityPlayer)null, 1009, new BlockPos((int)this.field_179469_a.posX, (int)this.field_179469_a.posY, (int)this.field_179469_a.posZ), 0);
+                        this.field_179469_a.worldObj.playAuxSFXAtEntity((EntityPlayer) null, 1009, new BlockPos((int) this.field_179469_a.posX, (int) this.field_179469_a.posY, (int) this.field_179469_a.posZ), 0);
 
-                        for (int i = 0; i < 1; ++i)
-                        {
-                            EntitySmallFireball entitysmallfireball = new EntitySmallFireball(this.field_179469_a.worldObj, this.field_179469_a, d1 + this.field_179469_a.getRNG().nextGaussian() * (double)f, d2, d3 + this.field_179469_a.getRNG().nextGaussian() * (double)f);
-                            entitysmallfireball.posY = this.field_179469_a.posY + (double)(this.field_179469_a.height / 2.0F) + 0.5D;
+                        for (int i = 0; i < 1; ++i) {
+                            EntitySmallFireball entitysmallfireball = new EntitySmallFireball(this.field_179469_a.worldObj, this.field_179469_a, d1 + this.field_179469_a.getRNG().nextGaussian() * (double) f, d2, d3 + this.field_179469_a.getRNG().nextGaussian() * (double) f);
+                            entitysmallfireball.posY = this.field_179469_a.posY + (double) (this.field_179469_a.height / 2.0F) + 0.5D;
                             this.field_179469_a.worldObj.spawnEntityInWorld(entitysmallfireball);
                         }
                     }
                 }
 
                 this.field_179469_a.getLookHelper().setLookPositionWithEntity(entitylivingbase, 10.0F, 10.0F);
-            }
-            else
-            {
+            } else {
                 this.field_179469_a.getNavigator().clearPathEntity();
                 this.field_179469_a.getMoveHelper().setMoveTo(entitylivingbase.posX, entitylivingbase.posY, entitylivingbase.posZ, 1.0D);
             }
@@ -276,4 +276,4 @@
             super.updateTask();
         }
     }
-}
+}
